From eb74f42c78f3d2e96f3bb80a30103e00ccdddbad Mon Sep 17 00:00:00 2001
From: Steffen Trumtrar <s.trumtrar@pengutronix.de>
Date: Fri, 26 Mar 2021 10:15:57 +0100
Subject: [PATCH] litesdcard: frontend: add missing __init__.py

Without the init.py setuptools will not install the frontend dir.

Signed-off-by: Steffen Trumtrar <s.trumtrar@pengutronix.de>
---
 litesdcard/frontend/__init__.py | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 litesdcard/frontend/__init__.py

diff --git a/litesdcard/frontend/__init__.py b/litesdcard/frontend/__init__.py
new file mode 100644
index 000000000000..e69de29bb2d1
